vm/README.md

```diff
@@ -43,6 +43,32 @@ In our CI, we build it using:
 - `binutils`: `2.23.2`
 - `llvm`: `3.8` on linux; `4.0.1` on darwin
 
+---
+**NOTE:**
+
+On certain versions of MacOS you might run into build errors of the form:
+
+```
+'stdlib.h''stdio.h'  filefile  notnot  foundfound
```

Comment:
That looks like interleaved output. Can you please add `--serial` to the `build` command to get non-interleaved output.

Reply:
Done. PR updated with more clear error message.